### PR TITLE
mpls (fixing IESG comments of Magnus, Alvaro, Roman)

### PR DESCRIPTION
**Pull request by vb-balazs**
All comments were replied and fixed as per 8:00pm CET, 09-09-2020

### CODE REVIEW

File: mpls/draft-ietf-detnet-mpls.xml [hunk 1]
```diff
@@ -709,31 +708,27 @@ DFx = DetNet member flow x over a TE LSP
...
      <t>
-       After a DetNet service is identified for a received DetNet MPLS
-       packet, as described above, an implementation MUST check if POF
-       is configured for that DetNet service.  When configured, the
-       implementation MUST track the sequence number contained in
-       received d-CWs and MUST ensure that packets are processed in the
-       order indicated in the received d-CW sequence number field, which
-       may not be in the order the packets are received. As defined in
+	   After a DetNet service is identified for a received DetNet MPLS packet,
+	   as described above, if POF is configured for that DetNet service, packets
```
Comment (dfedyk) [inline mpls/draft-ietf-detnet-mpls.xml:717]: Is it "if a POF" ?  if a Packet Ordering Function vs if Packet Ordering Function

File: mpls/draft-ietf-detnet-mpls.xml [hunk 2]
```diff
@@ -1395,18 +1390,20 @@ DFx = DetNet member flow x over a TE LSP
        network node within the boundary of the DetNet domain.
     </t>
     <t>
-    The primary consideration for the DetNet data plane is to maintain
+    Additional consideration for the DetNet data plane is to maintain
```
Comment (dfedyk) [inline mpls/draft-ietf-detnet-mpls.xml]: An additional vs Additional

File: mpls/draft-ietf-detnet-mpls.xml (hunk 1 above)
Comment (vb-balazs) [reply mpls/draft-ietf-detnet-mpls.xml:717]: My favorite is "if POF", I will keep it.

File: mpls/draft-ietf-detnet-mpls.xml (hunk 2 above)
Comment (vb-balazs) [reply mpls/draft-ietf-detnet-mpls.xml]: OK. I will update.

**Review by louberger — APPROVED** [review]

**Pull request merged**